src/lv_misc/lv_mem.c

```diff
@@ -140,10 +140,9 @@ void _lv_mem_init(void)
 void _lv_mem_deinit(void)
 {
 #if LV_MEM_CUSTOM == 0
-    _lv_memset_00(work_mem, (LV_MEM_SIZE / sizeof(MEM_UNIT)) * sizeof(MEM_UNIT));
```

Comment:
Why is it removed?

Reply:
Here are several reasons:
1. _lv_mem_init doesn't zero memory too(especially when LV_MEM_ADR != 0)
2. it will take a long time to zero the huge memory block
3. memory manager don't care the playload value except lv_mem_ent_t field

Reply:
Sounds reasonable. Thanks!